api/MediaKeySession.json

```diff
@@ -29,10 +29,10 @@
             "version_added": true
           },
           "safari": {
-            "version_added": null
+            "version_added": true
```

Comment:
Yep, `window.MediaKeySession` is in Safari 13.1 at least. We can narrow down the range later.